R/navbar.R

```diff
@@ -16,8 +16,9 @@ data_navbar <- function(pkg = ".", depth = 0L) {
     depth = depth,
     bs_version = pkg$bs_version
   )
+  brand <- list(brand = navbar_brand(pkg, depth))
```

Comment:
it's a bit misleading reg CSS to call it brand since only the package name has the brand class (in BS5)